src/databricks/labs/ucx/config.py

```diff
@@ -11,6 +11,7 @@ class WorkspaceConfig:  # pylint: disable=too-many-instance-attributes
     __version__ = 2
 
     inventory_database: str
+    inventory_catalog: str = "hive_metastore"
```

Comment:
revert this change. we'll always have data in HMS (when we run run on existing workspaces)